sirius/plugins/org.polarsys.kitalpha.sirius.typeinterpreter/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Kitalpha Sirius Type Interpreter
+Bundle-SymbolicName: org.polarsys.kitalpha.sirius.typeinterpreter;singleton:=true
+Bundle-Version: 5.2.0.qualifier
+Bundle-Activator: org.polarsys.kitalpha.sirius.typeinterpreter.TypeInterpreterPlugin
+Bundle-Vendor: THALESGROUP
```

Comment:
%providerName in a .properties.
In kitalpha its seems to be www.polarsys.org
https://github.com/eclipse/kitalpha/blob/master/cadence/plugins/org.polarsys.kitalpha.cadence.ui/META-INF/MANIFEST.MF#L7